pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -160,22 +160,25 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 			attr.name, attr.allocated.String(), attr.request.String(), attr.inqueue.String(), attr.elastic.String())
 	}
 
+	// Record metrics
```

Comment:
How about put it at where the Deserved has been calculated? so that to include Deserved metrics.

Reply:
Deserved metrics are calculated in a loop，it's not convenient to put them together.